backend/api/projects.py

```diff
@@ -248,11 +248,11 @@ def add_comment(project_id):
                   or 404 if project is not found
     """
     try:
-        manifest = g.projects.find({'_id': project_id}, {'comments': 1, '_id': 0}).sort(
+        comments = g.projects.find({'_id': project_id}, {'comments': 1, '_id': 0}).sort(
                    key=lambda x: x['datetime'], reverse=True)
-        if manifest is None:
+        if comments is None:
             raise ApiException("Project not found", 404)
-        return jsonify(manifest['comments'])
+        return jsonify(comments)
```

Comment:
I dont think this is the intended return. The (list of) comments should be returned, not the entire json I think?